### ISSUE TITLE
Steam cancels KDE logout

### ISSUE THREAD

**Issue by DrDoctor13** [ValveSoftware/steam-for-linux#5094]
#### Your system information

* Steam client version (build number or date): July 17, 2017
* Distribution (e.g. Ubuntu): Manjaro
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Trying to logout or otherwise end the session in KDE will cause Steam to open a window for a split-second then exit, but it will cancel the logout. Plasma Workspace will then show a notification saying "Logout was canceled by ''".

#### Steps for reproducing this issue:

1. Log in to a Plasma session
2. Open Steam
3. Try to end the session through logging out, restarting, or shutting down.

**Comment by kisak-valve** [comment]
Hello @DrDoctor13, this issue is currently being tracked at #3301. Closing as a duplicate.

**Comment by Jibodeah** [comment]
This is **not** a duplicate of that issue. The problem isn't that shutdown is slower when steam is running. The problem is that a request to logout, restart, or shutdown is **cancelled**. Steam also closes at the same time.

Here's some logs from steam starting from when a logout is prompted.

```
process 20610: arguments to dbus_connection_ref() were incorrect, assertion "connection->generation == _dbus_current_generation" failed in file ../../dbus/dbus-connection.c line 2656.
This is normally a bug in some application using the D-Bus library.
process 20610: arguments to dbus_connection_unref() were incorrect, assertion "connection->generation == _dbus_current_generation" failed in file ../../dbus/dbus-connection.c line 2794.
This is normally a bug in some application using the D-Bus library.
```
[above two messages repeat numerous times]
```
[2017-11-08 02:32:52] Startup - updater built Nov  7 2017 13:43:41
[2017-11-08 02:32:52] Opted in to client beta 'publicbeta' via beta file
You are in the 'publicbeta' client beta.
[2017-11-08 02:32:52] Verifying installation...
[2017-11-08 02:32:52] Verification complete
[2017-11-08 02:33:10] Shutdown
```

I'm running Fedora 26 using KDE Plasma 5.10.5.

**Comment by poppyfields** [comment]
Same issue (logout cancelled!) on Archlinux KDE plasma 5.11.3.

**Comment by digibucc** [comment]
Same issue (logout cancelled!) on Archlinux KDE plasma 5.11.3.

**Comment by TravMurav** [comment]
Same issue (logout cancelled!) on Kubuntu 17.10 KDE plasma 5.10.5.

**Comment by Kran1x** [comment]
Same issue (logout cancelled!) on Manjaro KDE Plasma 5.11.4.

**Comment by zatricky** [comment]
Original description of #3301 is re slowdown - but the topic changed. Pretty much all +1s there are regarding logout/shutdown being cancelled.

**Comment by xeetsh** [comment]
Same issue! On Archlinux with plasma 5.11.5.

**Comment by programaths** [comment]
Software
OS: Kubuntu 17.10
KDE Plasma: 5.10.5
KDE Frameworks= 5.38.0
Qt : 5.9.1
Kernel: 4.13.0-32-generic
OS Type: 64-bit

HW (HP DV7)
Processor: Intel Core i7-2670QM @ 2.20GHz
Memory: 8GiB

**Comment by Dinoraptor101** [comment]
Same issue here
Manjaro Linux
KDE Plasma 5.11.5
KDE Frameworks 5.42.0
QA Vesion: 5.10.0
Kerlen: 4.14.14-1-MANJARO
64 bit archetecture
CPU: Intel I5-2500k

**Comment by danban359** [comment]
Same issue on Kubuntu, but the application that cancels the logout is Viber.

**Comment by zatricky** [comment]
If it is Viber causing a problem, not sure how posting about it here is going to help?

According to #3301 the issue is fixed in Beta.

**Comment by programaths** [comment]
@zatricky It may help you if you can cross analyse the issue. Maybe it is related to something both apps do!

**Comment by freoff** [comment]
hipchat4 cancels logout too.

**Comment by lucarillosi** [comment]
Same issue
Manjaro Linux
KDE Plasma 5.12.2

**Comment by lucarillosi** [comment]
Seems to have been solved as of version 1521502351
[Changelog](http://store.steampowered.com/news/38412/)
